### PR TITLE
Add spinners while waiting for response from server

### PR DESCRIPTION
**Pull request by diastremskii**

### CODE REVIEW

File: build/components/cic-main/element.html
Comment (diastremskii) [inline build/components/cic-main/element.html]: Wow, now that looks bad. I thought what at least comments are stripped.

**Review by kashesandr — COMMENTED** [review]
@TheBeastOfCaerbannog Generally looks good! Except those extra lines of css and... Actually I think the spinner looks a bit sloppy. But with the spinner is better that without, so I'll merge if you don't have possibility to find a better visual implementation.

**Comment by kashesandr** [comment]
@TheBeastOfCaerbannog also there are some conflicts after I merged #14

**Review by kashesandr — APPROVED** [review]

**Comment by diastremskii** [comment]
Yeeeah, looks pretty bad. I chose between three options: paper-spinner, spin.js and using GIF. So I thought what paper element will be more native and extra CSS is better than extra JS :)
What do you think would be a better way: throw away paper-spinner and try something different or customise it?
I guess moving spinner inside buttons should make it look better.

**Comment by diastremskii** [comment]
So, now build does not include spinners. Maybe I shouldn't include build in PRs to avoid conflicts

**Comment by diastremskii** [comment]
out
Looks a bit better?

**Comment by kashesandr** [comment]
@TheBeastOfCaerbannog looks much better! Will have a look the changes on this week! Thank you!

**Review by kashesandr — APPROVED** [review]

**Pull request merged**
